Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2
# CoA Fixes

## Changelog:
- Fixed issues where Loa, Geo, and Chi religious followers weren't using their unique CoAs.


# How to test:
Enter the game, check which CoAs have trolls, centaurs, and pandaren, tell your opinion.
